app/controllers/alerts_list_controller.rb

```diff
@@ -16,6 +17,15 @@ def index
     redirect_to :action => 'show'
   end
 
+  def class_icons
+    res = {}
+    icons.each do |k, _val|
+      file_name = "100/#{k.to_s.underscore}.png"
```

Comment:
That would be nice and allow me to simplify code a bit, is decorate available for classes too or only instances? I can't seem to use it